src/docs/asciidoc/community/index.adoc

```diff
@@ -64,8 +64,40 @@ Vulnerabilities can be reported publicly in the same way.
 
 include::workflow.adoc[leveloffset=+2]
 
+=== Renovate PR
 
+Renovate automates dependency upgrades, proposing PRs for updates. Before merging:
 
+Devtools:
+
+- No specific checks are generally needed.
+- Exceptions: 
+
+    * For major linter upgrades (e.g., X.X.X to X.Y.X), perform a local test to ensure the linter continues to function correctly.
```

Comment:
major upgrades : do you mean X.X.X to Y.X.X ? 
I propose : "continues to function correctly." => "is still correctly."